app/celer-sim/Transporter.cc

```diff
@@ -158,6 +160,9 @@ auto Transporter<M>::operator()(SpanConstPrimary primaries) -> TransporterResult
         record_step_time();
     }
 
+    auto counters = copy_to_host(stepper_->state_ref().init.track_counters);
+    result.num_tracks = std::accumulate(
+        counters.data().get(), counters.data().get() + counters.size(), 0);
```

Comment:
```suggestion
        counters.data().get(), counters.data().get() + counters.size(), size_type(0));
```
otherwise it's accumulating a signed int and then implicitly casting the result to a size_type.